src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+
```

Comment:
```suggestion
   private String[] colors = { "white", "black".... };
```
we don't need to create new array and fill it with values per each method call